openedx/core/djangoapps/theming/storage.py

```diff
@@ -4,21 +4,26 @@
 """
 import os.path
 import posixpath
+import re
 
+import django
 from django.conf import settings
 from django.contrib.staticfiles.finders import find
 from django.contrib.staticfiles.storage import CachedFilesMixin, StaticFilesStorage
 from django.utils._os import safe_join
-from django.utils.six.moves.urllib.parse import unquote, urlsplit  # pylint: disable=no-name-in-module, import-error
-from pipeline.storage import PipelineMixin
-
+from django.utils.six.moves.urllib.parse import (  # pylint: disable=no-name-in-module, import-error
+    unquote,
+    urldefrag,
+    urlsplit
+)
 from openedx.core.djangoapps.theming.helpers import (
     get_current_theme,
     get_project_root_name,
     get_theme_base_dir,
     get_themes,
     is_comprehensive_theming_enabled
 )
+from pipeline.storage import PipelineMixin
```

Comment:
We normally try to keep third-party imports above repo-local ones: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#imports-order